templates/aws/pro.html

```diff
@@ -0,0 +1,267 @@
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Open source security</h3>
+          <p class="p-matrix__desc">Ubuntu Pro adds security coverage for the most important open source applications like Apache Kafka, NGINX, MongoDB, Redis and PostgreSQL.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">FIPS and CC-EAL2 certification</h3>
+          <div class="p-matrix__desc">
+            <p>Certified components to enable operating environments under compliance regimes like FedRAMP, HIPAA, PCI and ISO.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">CIS and DISA STIG profiles</h3>
+          <p class="p-matrix__desc">For companies looking to leverage industry benchmarks for hardening, Ubuntu Pro makes two leading profiles available out of the box.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">AWS-integrated</h3>
+          <p class="p-matrix__desc">Starting in Q1 2020, Ubuntu Pro will include integrations with AWS services including Security Hub, CloudWatch and CloudTrail.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Landscape included</h3>
+          <div class="p-matrix__desc">
+            <p>Landscape provides a complete solution for the patching, compliance reporting and lifecycle management of an Ubuntu cloud fleet via a web UI or API.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">10-year lifetime</h3>
+          <div class="p-matrix__desc">
+            <p>Canonical backs Ubuntu Pro for 10 years<a href="#index-1">*</a>, ensuring security updates are available throughout, with a guaranteed upgrade path.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Priced for the cloud</h3>
+          <div class="p-matrix__desc">
+            <p>Ubuntu Pro pricing tracks the underlying EC2 cost, starting at 20% for the smallest t3.nano instances, and ramping down to less than 1%.</p>
+          </div>
+        </div>
+      </li>
+    </ul>
+    <p id="index-1"><small>* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years.</small></p>
+    <p><a href="/security">Learn more about security for Ubuntu&nbsp;&rsaquo;</a></p>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
```

Comment:
```suggestion
  <div class="u-fixed-width">
```

Reply:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.